lib/very_good_infinite_list.dart

```diff
@@ -417,14 +415,14 @@ class _ListController<T> extends ValueNotifier<_ListState<T>> {
 }
 
 class _Debouncer {
-  _Debouncer({Duration delay}) : _delay = delay ?? _kDebounceDuration;
+  _Debouncer({Duration? delay}) : _delay = delay ?? _kDebounceDuration;
 
-  final Duration _delay;
-  Timer _timer;
+  final Duration? _delay;
```

Comment:
```suggestion
  final Duration _delay;
```

Reply:
Oh yes, this never gonna be null.
sorry for the silly mistakes.